README.rst

```diff
@@ -75,3 +86,83 @@ Most of the cron jobs and the jobqueue daemon are running.  The push connector
 is not currently configured, nor is it configured to talk to Pulse.
 
 It is configured to use memcached as well.
+
+
+Docker Container
+================
+
+This repository is also a runnable docker container.
+
+Container Arguments
+-------------------
+
+Currently, the entry point takes a single command argument.
+This can be **httpd** or **shell**.
+
+httpd
+    This will start apache listening for connections on ``$PORT``
+shell
+    This will start an interactive shell in the container. Useful for debugging.
+
+
+Environmental Variables
+-----------------------
+
+PORT
+  This must be a value >= 1024. The httpd will listen on this port for incoming
+  plain-text HTTP connections.
+
+BMO_db_driver
+  What SQL database to use. Default is mysql. List of supported databases can be
+  obtained by listing Bugzilla/DB directory - every module corresponds to one
+  supported database and the name of the module (before ".pm") corresponds to a
+  valid value for this variable.
+
+BMO_db_host
+  The DNS name or IP address of the host that the database server runs on.
+
+BMO_db_name
+  The name of the database
+
+BMO_db_user
+  The database user to connect as.
+
+BMO_db_pass
+  The password for the user above
+
+BMO_site_wide_secret
+  This secret key is used by your installation for the creation and
+  validation of encrypted tokens. These tokens are used to implement
+  security features in Bugzilla, to protect against certain types of attacks.
+  It's very important that this key
+  is kept secret. It also must be very long.
+
+BMO_inbound_proxies
+  This is a list of IP addresses that we expect proxies to come from.
```

Comment:
bmo used to run in a way that meant you could hit it directly from the outside world.